.github/workflows/benchmark.yml

```diff
@@ -44,6 +44,13 @@ jobs:
         if: steps.cache-deps.outputs.cache-hit == 'true'
       # </common-build>
 
+      # Restore performance downloaded states
```

Comment:
what do we cache this for?

Reply:
The states downloaded from Infura for performance benchmarks of epoch processing, so we don't have to download them every time